SPECS/libdrm/libdrm.spec

```diff
@@ -207,125 +199,144 @@ mkdir -p %{buildroot}%{_docdir}/libdrm
 %{_libdir}/pkgconfig/libdrm_intel.pc
 %endif
-
 %if %{with radeon}
-%{_includedir}/libdrm/radeon_*.h
+%{_includedir}/libdrm/radeon_{bo,cs,surface}*.h
 %{_includedir}/libdrm/r600_pci_ids.h
 %{_libdir}/libdrm_radeon.so
 %{_libdir}/pkgconfig/libdrm_radeon.pc
 %endif
-
 %if %{with amdgpu}
 %{_includedir}/libdrm/amdgpu.h
 %{_libdir}/libdrm_amdgpu.so
 %{_libdir}/pkgconfig/libdrm_amdgpu.pc
 %endif
-
 %if %{with nouveau}
 %{_includedir}/libdrm/nouveau/
 %{_libdir}/libdrm_nouveau.so
 %{_libdir}/pkgconfig/libdrm_nouveau.pc
 %endif
-
 %if %{with omap}
 %{_includedir}/libdrm/omap_*.h
 %{_includedir}/omap/
 %{_libdir}/libdrm_omap.so
 %{_libdir}/pkgconfig/libdrm_omap.pc
 %endif
-
 %if %{with exynos}
 %{_includedir}/libdrm/exynos_*.h
 %{_includedir}/exynos/
 %{_libdir}/libdrm_exynos.so
 %{_libdir}/pkgconfig/libdrm_exynos.pc
 %endif
-
 %if %{with freedreno}
 %{_includedir}/freedreno/
 %{_libdir}/libdrm_freedreno.so
 %{_libdir}/pkgconfig/libdrm_freedreno.pc
 %endif
-
 %if %{with tegra}
 %{_includedir}/libdrm/tegra.h
 %{_libdir}/libdrm_tegra.so
 %{_libdir}/pkgconfig/libdrm_tegra.pc
 %endif
-
 %if %{with vc4}
 %{_includedir}/libdrm/vc4_*.h
 %{_libdir}/pkgconfig/libdrm_vc4.pc
 %endif
-
 %if %{with etnaviv}
 %{_includedir}/libdrm/etnaviv_*.h
 %{_libdir}/libdrm_etnaviv.so
 %{_libdir}/pkgconfig/libdrm_etnaviv.pc
 %endif
-
+%{_includedir}/libsync.h
+%{_includedir}/xf86drm.h
+%{_includedir}/xf86drmMode.h
+%if %{with man_pages}
+%{_mandir}/man3/drm*.3*
+%{_mandir}/man7/drm*.7*
+%endif
 
 %if %{with install_test_programs}
 %files -n drm-utils
+%if %{with amdgpu}
+%{_bindir}/amdgpu_stress
+%endif
 %{_bindir}/drmdevice
+%if %{with etnaviv}
 %exclude %{_bindir}/etnaviv_*
+%endif
+%if %{with exynos}
 %exclude %{_bindir}/exynos_*
-%{_bindir}/kms-steal-crtc
-%{_bindir}/kms-universal-planes
-
-%if %{with libkms}
-%{_bindir}/kmstest
 %endif
-
+%if %{with tegra}
+%exclude %{_bindir}/tegra-*
+%endif
 %{_bindir}/modeprint
 %{_bindir}/modetest
 %{_bindir}/proptest
 %{_bindir}/vbltest
 %endif
 
-
 %changelog
-* Fri Apr 22 2022 Olivia Crain <oliviacrain@microsoft.com> - 2.4.102-5
+* Mon May 15 2023 Hideyuki Nagase <hideyukn@microsoft.com> - 2.4.114-1
+- Initial CBL-Mariner import from Fedora 39 (license: MIT).
+- Merge libdrm.spec from previous Mariner version contains below changes
 - Remove explicit pkgconfig provides that are now automatically generated by RPM
-
-* Mon Aug 30 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 2.4.102-4
-- Removing BR on 'marinerui-rpm-macros'. Using macros from the build env.
-
-* Fri Dec 18 2020 Pawel Winogrodzki <pawelwi@microsoft.com> - 2.4.102-3
-- Initial CBL-Mariner import from Fedora 33 (license: MIT).
 - Added an explicit 'pkgconfig(libdrm)' for 'libdrm-devel'.
-- Added build-time dependency on 'marinerui-rpm-macros'.
 - Added the "LICENSE.PTR" file.
 - Removed manual pages.
 - Replaced ldconfig scriptlets with explicit calls to ldconfig.
 - Removed sections related to %%bcond_with for 'cairo_tests' and 'freedreno_kgsl'.
```

Comment:
For changes to a spec we already had in our repo, please leave the old changelog entries and just mention that for your update you've used Fedora X for guidance. Look for `spec (license: MIT) for guidance` across the specs to see what I have in mind.

Reply:
Yes, this is what I wonder, since Mariner's change after fork from original Fedora, has own history and timeline/version can't be sorted/aligned. Do you mean I don't need to incorporate the new history (below line 304 ~ 339) from Fedora but just add that one line saying which version of Fedora used for guidance?

Reply:
Updated